public/app/plugins/datasource/prometheus/types.ts

```diff
@@ -34,6 +35,7 @@ export interface PromOptions extends DataSourceJsonData {
   exemplarTraceIdDestinations?: ExemplarTraceIdDestination[];
   prometheusType?: PromApplication;
   prometheusVersion?: string;
+  cacheLevel?: PrometheusCacheLevel;
```

Comment:
Can we make this required? Or better I must ask is there a reason behind we make this `optional`?